networking/src/Models/AWS/AWSTeamAnswer.ts

```diff
@@ -0,0 +1,14 @@
+import { ConfidenceLevel } from "../../AWSMobileApi";
+
+export type AWSTeamAnswer = {
+  id: string
+  questionId?: string | null
+  isChosen: boolean
+  isTrickAnswer: boolean
+  text?: string | null
+  awsAnswerContents?: string | null
+  createdAt?: string
+  updatedAt?: string
```

Comment:
These shouldn't be nullable for this model and other `AWSXXX` models

Reply:
fixed